powerhome/playbook
# [PLAY-314]- Added Rails React Stack Example

#### Screens

![Screen Shot 2022-09-28 at 2 32 43 PM](https://user-images.githubusercontent.com/73710701/193078390-40f68387-ac2c-46d1-b8f4-5bc8bf610a03.png)

#### Breaking Changes

No breaking changes, PR adds example app for Rails+React stack using react-rails

#### Runway Ticket URL

[Runway Story](https://nitro.powerhrg.com/runway/backlog_items/PLAY-314)

#### How to test this

Cd into examples/rails-react-example and run yarn install than bin/rails s to start server. Navigate to localhost:3000/pages to see example page

#### Checklist:
